src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static void printGreeting() {
+        String logo = "    *******     *******   *******       **\n"
+                + "    /**////**   **/////** /**////**     ****\n"
+                + "    /**    /** **     //**/**   /**    **//**\n"
+                + "    /**    /**/**      /**/*******    **  //**\n"
+                + "    /**    /**/**      /**/**///**   **********\n"
+                + "    /**    ** //**     ** /**  //** /**//////**\n"
+                + "    /*******   //*******  /**   //**/**     /**\n"
+                + "    ///////     ///////   //     // //      //\n";
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println(logo + "\n    Kon'nichiwa! Doraemon desu.\n    What can I do for you?");
```

Comment:
Very creative use of a personalised interface :)

Reply:
Thanks Jun Hao, appreciate your encouragement!